frontend/src/components/QuestionSug/style.jsx

```diff
@@ -1,22 +1,42 @@
 import styled from "styled-components";
 
 export default styled.div`
-  div {
-    text-align: center;
+  .smileyImg {
+    display: flex;
+    justify-content: center;
   }
-
   h1,
   h2,
   h3 {
     text-align: center;
     padding-top: 1rem;
   }
-
-  .smileyImg {
-    display: flex;
-    padding: 2rem;
+  Img {
```

Comment:
`Img` should be `img` (the html tag) *or* `.Img` (a class name (albeit a bad one))